packages/vizbuilder/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@datawheel/vizbuilder",
+  "description": "Visualization explorer for data retrieved from mondrian-rest.",
+  "version": "0.1.0",
+  "main": "src/index.js",
+  "author": "Francisco Abarzúa & Carlos Navarrete",
+  "license": "MIT",
+  "typings": "index.d.ts",
```

Comment:
The *.d.ts files are typescript files, but their function is to provide typing information about functions for javascript modules and libraries. Since javascript doesn't work with typing like typescript does, module owners can create a *.d.ts file and add it to their package.json, so VisualStudio Code and other editors are able to parse and provide static typing verification.

Some editors are able to parse JSDoc and do the same (VSCode actually can), but this functionality is limited when you try to import a module from a npm package. This file can surpass that limitation.

Reply:
Removed in ce3b1ae369befac12d261191ad069dbe5bf8c39a, though I think we should add it at some point in the future.